diff --git a/drivers/amlogic/amports/esparser.c b/drivers/amlogic/amports/esparser.c
index cbb2fff..df87072 100755
--- a/drivers/amlogic/amports/esparser.c
+++ b/drivers/amlogic/amports/esparser.c
@@ -295,11 +295,6 @@ s32 esparser_init(struct stream_buf_s *buf)
         WRITE_MPEG_REG(RESET1_REGISTER, RESET_PARSER);
 
         /* TS data path */
-#ifndef CONFIG_AM_DVB
-        WRITE_MPEG_REG(FEC_INPUT_CONTROL, 0);
-#else
-        tsdemux_set_reset_flag();
-#endif
         CLEAR_MPEG_REG_MASK(TS_HIU_CTL, 1 << USE_HI_BSF_INTERFACE);
         CLEAR_MPEG_REG_MASK(TS_HIU_CTL_2, 1 << USE_HI_BSF_INTERFACE);
         CLEAR_MPEG_REG_MASK(TS_HIU_CTL_3, 1 << USE_HI_BSF_INTERFACE);
diff --git a/drivers/amlogic/amports/rmparser.c b/drivers/amlogic/amports/rmparser.c
index fbc364f..d4fabea 100755
--- a/drivers/amlogic/amports/rmparser.c
+++ b/drivers/amlogic/amports/rmparser.c
@@ -80,11 +80,6 @@ s32 rmparser_init(void)
     WRITE_MPEG_REG(RESET1_REGISTER, RESET_PARSER);
 
     /* TS data path */
-#ifndef CONFIG_AM_DVB
-    WRITE_MPEG_REG(FEC_INPUT_CONTROL, 0);
-#else
-    tsdemux_set_reset_flag();
-#endif
     CLEAR_MPEG_REG_MASK(TS_HIU_CTL, 1 << USE_HI_BSF_INTERFACE);
     CLEAR_MPEG_REG_MASK(TS_HIU_CTL_2, 1 << USE_HI_BSF_INTERFACE);
     CLEAR_MPEG_REG_MASK(TS_HIU_CTL_3, 1 << USE_HI_BSF_INTERFACE);
